win/libsass.sln

```diff
@@ -1,10 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
```

Comment:
I *think* 2015 sln file works with 2013 and 2012, but not the lower ones. Besides, from command line perspective, `msbuild` adapts to the installed toolchain.

Now that things are moving fast with the advent of VS Code for windOS and Unices, the previewed full VS for Mac and previewed VS2017, I honestly see no point in sticking with 2013 when we still can provide *better* support from XP-10 and all related server editions with latest CRT/CPP toolchains (brining dozens of bugfixes in STL and standard conformance). If there is a bug refraining us to moving forward, like any other piece of software, lets report the issue upstream, track it in our repo and keep the dice rolling forward when we are no backward compatibility trade-offs. Officially VC team hasn't even dropped XP support yet (released: 24.04.2001).

My €0.2 :)